src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -2130,7 +2130,8 @@ function cleanRenderedMarkdown(element: Node): void {
 }
 
 function fixSettingLinks(text: string, linkify = true): string {
-	return text.replace(/`#([^#]*)#`|'#([^#]*)#'/g, (match, settingKey) => {
+	return text.replace(/`#([^#]*)#`|'#([^#]*)#'/g, (match) => {
+		const settingKey = match.substring(2, match.length - 2);
```

Comment:
It's fine for a candidate but it might be more clear to do this differently - you could write the regex like `(\`|')#...`, or even just have parameters like `(match, settingKey1, settingKey2) => ... settingKey = settingKey1 ?? settingKey2...`